lib/gen_socket_client/transport/web_socket_client.ex

```diff
@@ -2,6 +2,11 @@ defmodule Phoenix.Channels.GenSocketClient.Transport.WebSocketClient do
   @moduledoc """
   Websocket transport powered by the [websocket_client](https://github.com/sanmiguel/websocket_client)
   library.
+
+  Supported transport options:
+
+    - `keepalive` - Interval in which a ping message is sent to the server to keep the connection alive.
+      Defaults to 30 seconds.
```

Comment:
Well, if it's missing, then the default is used. But you have the point about > 60 seconds.